kevm-pyk/src/kevm_pyk/utils.py

```diff
@@ -64,9 +67,10 @@ def get_apr_proof_for_spec(  # noqa: N802
     return apr_proof
 
 
-def kevm_apr_prove(
+def kevm_prove(
     kprove: KProve,
-    proof: APRProof | APRBMCProof,
+    cfgid: str,
+    proof: Proof,
```

Comment:
Do we actually need to pass in `cfgid` again as well? Can we just use `proof.id` below wherever `cfgid` is used?

Reply:
Removed it.